setup.py

```diff
@@ -116,22 +117,25 @@ def get_blosc2_directories():
         raise EnvironmentError(
             "Cannot find neither the c-blosc2 package nor the "
             "python-blosc2 wheel")
-    version = blosc2.__version__
-    basepath = Path(os.path.dirname(blosc2.__file__))
-    recinfo = basepath.parent / f'blosc2-{version}.dist-info' / 'RECORD'
+
+    files = import_metadata_files('blosc2')
```

Comment:
This will make the life very hard for packagers.
Please make the setup script compatible with a scenario in which cblosc2 is provided as system library.

Conda could be used in CI to test this scenario.

Reply:
But wasn't it the same case before? Previously it directly looked at the record file. All this change is doing is using `importlib.metadata` to inspect the record instead of us doing it manually. Unless I'm missing something!?

Reply:
Do you mean for a case in which `cblosc2` is installed but not `blosc2`? I think it is not currently supported even before my changes!?

Reply:
Ah sorry, I'm commenting on what I see in the diff.
There could be some pre-existing issue but please consider that all the blosc2 part is quite new.
In any case we should try to fix.